docs/getting_started.rst

```diff
@@ -240,7 +242,7 @@ Hosting
 
 If you are hosting Red on your personal computer, you will soon notice that
 if you close the window or if you shut down you computer, Red will be offline.
-She needs an environment to work and respond.
+It needs an environment to work and respond.
 
 You can try to host Red somewhere he will always be online, like on a virtual
```

Comment:
You changed a "she" to "it", but didn't change the "he". After the discussion in AC, both of these should probably be she.